code/modules/mob/living/carbon/human/species_types/kepori.dm

```diff
@@ -17,10 +17,10 @@
 	species_clothing_path = 'icons/mob/clothing/species/kepori.dmi'
 	species_eye_path = 'icons/mob/species/kepori/kepori_eyes.dmi'
 	heatmod = 0.67
-	coldmod = 1.5
-	brutemod = 1.5
-	burnmod = 1.5
-	speedmod = -0.25
+	// coldmod = 1.5
```

Comment:
This is a temperature damage modifier like heatmod, and probably doesn't need to be changed

Reply:
yeah fair, must have caught it on accident